src/scjsv/core.clj

```diff
@@ -50,10 +71,38 @@
     :else (throw (Exception. (str "Don't know how to convert " (pr-str value)
                                   " into a JsonSchemaFactory.")))))
 
+
 ;;
 ;; Public API
 ;;
 
+(defn- build-validator
+  "Returns a validator (a single arity fn).
+  Schema can be given either as a JSON String or a Clojure Map.
+
+  To configure the validator, you can pass a JsonSchemaFactory instance or a
+  options map as the second parameter. See scjsv.core/validator docstring for
+  the options.
+  `->json-node` is the function which will be applied to datum to transform them into
+  a JsonNode"
+  [schema json-schema-factory ->json-node]
+  (comp (partial validate (->json-schema schema (->factory (or json-schema-factory
+                                                     (build-factory {})))))
+      ->json-node))
+
+(defn json-reader-validator
+  "Returns a JSON string validator (a single arity fn).
```

Comment:
Could you change this docstring so that it's clear that it takes a Reader and not a string?